e2e/angular-extensions/src/angular-library.test.ts

```diff
@@ -139,10 +139,12 @@ describe('Angular Package', () => {
       it('empty test to make jest happy', () => {});
 
       // These fail with pnpm due to incompatibilities with ngcc for buildable libraries.
-      if (
-        getSelectedPackageManager() !== 'pnpm' ||
-        testConfig === 'publishable'
-      ) {
+      // therefore switch to yarn
+      const previousPackageRunner = process.env.SELECTED_PM;
+      if (getSelectedPackageManager() === 'pnpm') {
+        process.env.SELECTED_PM = 'yarn';
+      }
```

Comment:
Does this run before the `beforeEach`?

I wonder if it would be better to make the `newProject` to allow passing a package manager for cases like this. It'd be more readable and easier to reason about.

Reply:
I think we'd still want to allow npm though. We only want to switch to Yarn when it's pnpm that's running the test